src/Enzyme.jl

```diff
@@ -989,5 +989,7 @@ end
     mapreduce(LinearAlgebra.adjoint, vcat, rows)
 end
 
+# Precompile the init
+# Compiler.__init__()
```

Comment:
Is this meant to be commented out?

Reply:
Oh, hah.. that was for my benchmark without this